### ISSUE TITLE
On refresh, either the handle function doesn't recognize the cookie or the handle function doesn't run at all.

### ISSUE THREAD

**Issue by saym97** [geoffrich/sveltekit-azure-swa-demo#2]
How to reproduce the issue:
    Go to Todos route and add some tasks. Then refresh the page. All of the previous task should be gone and new user_id is assigned in the cookie, which means the request header when refreshing lost the user_id cookie.

**Comment by geoffrich** [comment]
What browsers have you seen this issue in?

**Comment by geoffrich** [comment]
For me, I can consistently repro this on Firefox, but not Chrome (I'm on Windows). Not sure if it's an adapter, SK, or Azure problem -- will need to dig in a bit more.

**Comment by saym97** [comment]
yes, it's on the Firefox. Just tried it with brave browser, works great!! 
Lesson learnt : try your problem on different browsers before creating an issue.
 Thank you very much !!!

**Comment by geoffrich** [comment]
No worries! It's still an issue if it only happens on Firefox. Just wanted to make sure it matched what I was seeing.
